manifests/repo.pp

```diff
@@ -20,13 +20,16 @@
     notify  => Anchor['pulpcore::repo'],
   }
 
-  package { 'pulpcore-dnf-module':
-    ensure      => $dist_tag,
-    name        => 'pulpcore',
-    enable_only => true,
-    provider    => 'dnfmodule',
-    require     => File['/etc/yum.repos.d/pulpcore.repo'],
-    notify      => Anchor['pulpcore::repo'],
+  # Only EL8 has DNF modules
+  if $dist_tag == 'el8' {
```

Comment:
the tests still expect this on el9…

Reply:
Yes, I thought about fixing the other issues while I was at it but that it was filled with rabbits just to get an EL9 box within Vagrant.